components/builder-web/stylesheets/base/_colors.scss

```diff
@@ -47,11 +47,14 @@ $disabled: $medium-gray;
 
 $status-colors: (
   complete: $success,
+  completed: $success,
```

Comment:
😬 I'd rather not add things like 'completed` and `failed` when we have `complete`, `failure`. Is this just because the API is returning different strings in different situations? If so, we should fix this at the API and in the meantime, translate somehow, rather than propagate the inconsistency.

Reply:
That's exactly it. The API here returns "completed" and "failed". I could try to take a look and fix this in the API?

Reply:
IMO it'd be fine to file an issue on Builder with the request to change it, and in the meantime patch it on the front end somewhere close to where it's deserialized. Maybe in a reducer?